metadata-models/src/main/resources/entity-registry.yml

```diff
@@ -757,6 +757,17 @@ plugins:
           aspectName: 'schemaMetadata'
         - entityName: '*'
           aspectName: 'editableSchemaMetadata'
+    - className: 'com.linkedin.metadata.aspect.hooks.OwnershipOwnerTypes'
+      enabled: true
+      supportedOperations:
+        - CREATE
+        - UPSERT
+        - UPDATE
+        - RESTATE
+        - PATCH
```

Comment:
PATCH is technically not needed, since the mutator is applied after the application of the patch on the current object and at that point the changeType is an UPSERT

Reply:
not a blocker